app/src/state/transcribe.ts

```diff
@@ -65,8 +65,19 @@ export const startTranscription = createAsyncThunk<
   }
   dispatch(openTranscribing());
 
+  dispatch(setProgress(0));
+  dispatch(setState(TranscriptionState.CONVERTING));
+
   const fileName = basename(path);
-  const wavFileContent = await convertToWav(path);
+  let wavFileContent;
+  try {
+    wavFileContent = await convertToWav(path, (p) => dispatch(setProgress(p)));
```

Comment:
I think it is somewhat bad ux to restart the progressbar from 0. maybe we add old-school multiprogressbars?
or we just allocate 10% of the progress to the convert to wav?

Reply:
I also think multi-progress-bars would be the best way. But maybe let's add them in the next step?

Reply:
sounds like a plan :)

Reply:
I created #240 for that.